packages/rooks/src/hooks/useKeys.ts

```diff
@@ -134,6 +134,38 @@ function useKeys(
 
     return noop;
   }, [when, target, keysList, handleKeyDown, handleKeyUp]);
+
+  useEffect(() => {
+    if (typeof window !== "undefined") {
+      const originalAlert = window.alert;
```

Comment:
@qqpann While this is hacky, I can see how this will get the job done, can we add a cleanup to this effect, that will revert window.alert back to original alert when this component unmounts?

Reply:
This is a pretty awesome solution, but I am just feeling yuck looking at this, because normally we would expect this to be taken care of by native browser, but we are having to come up with a solution like this to patch the browser. 😓 

Also if we can somehow make this opt-in, that would be even more awesome. Perhaps a param, that when set, will run this window.alert stuff.

Reply:
OK. Made these changes.

Reply:
@qqpann Man you are on 🔥 . Amazing!